### ISSUE TITLE
where is the new loss function?

### ISSUE THREAD

**Issue by jahaur1** [qcw/CARD#2]
i don't find in exp.(train) and .sh

**Comment by KIKI0608** [comment]
Original i thought the authors should put all the loss functions including the SIGNAL DECAY-BASED LOSS FUNCTION in the folder of ***\CARD\long_term_forecast_l96\utils\losses.py**, but later i did find it that the authors implemented this new loss by adding some codes in the file 'exp_long_term_forecasting.py' from line 77 to line 81.
